LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/the-stylepropertymap/properties/user-select-expected.txt

```diff
@@ -1,14 +1,14 @@
 
-FAIL Can set 'user-select' to CSS-wide keywords: initial Invalid property user-select
-FAIL Can set 'user-select' to CSS-wide keywords: inherit Invalid property user-select
-FAIL Can set 'user-select' to CSS-wide keywords: unset Invalid property user-select
-FAIL Can set 'user-select' to CSS-wide keywords: revert Invalid property user-select
-FAIL Can set 'user-select' to var() references:  var(--A) Invalid property user-select
-FAIL Can set 'user-select' to the 'auto' keyword: auto Invalid property user-select
-FAIL Can set 'user-select' to the 'text' keyword: text Invalid property user-select
-FAIL Can set 'user-select' to the 'none' keyword: none Invalid property user-select
-FAIL Can set 'user-select' to the 'contain' keyword: contain Invalid property user-select
-FAIL Can set 'user-select' to the 'all' keyword: all Invalid property user-select
+PASS Can set 'user-select' to CSS-wide keywords: initial
+PASS Can set 'user-select' to CSS-wide keywords: inherit
+PASS Can set 'user-select' to CSS-wide keywords: unset
+PASS Can set 'user-select' to CSS-wide keywords: revert
+PASS Can set 'user-select' to var() references:  var(--A)
+FAIL Can set 'user-select' to the 'auto' keyword: auto assert_equals: expected "auto" but got "text"
```

Comment:
Hm... this behavior discrepancy is a bit concerning. We should figure out if we can fix this before unprefixing the property.

Reply:
I agree, we should try to fix it first. I started here, but never got around to finishing it: https://github.com/WebKit/WebKit/pull/36579

Reply:
Feel free to take over the PR if you wish.

> As this fail happens with the prefixed property, could we not work on unprefix this first and after work on this missing parameter?

I tried a first time, and I had to revert because of bug 242006 wasn't fixed, and that broke outlook.